apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -103,6 +102,10 @@ export function Chat({
       body: {
         id: params.threadId || isNewChat ? threadId : activeThread?.threadId,
         model: selectedModel,
+        chatbot: activeChatbot && activeChatbot?.categories?.length ? {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)